matchbox/models/fields.py

```diff
@@ -198,7 +201,11 @@ def lookup_value(self, lookup_type, value):
         return self.db_value(value)
 
     def db_value(self, value):
-        if not issubclass(value.__class__, self.ref_model):
+        if hasattr(value, "full_collection_name") and hasattr(value, "id"):
```

Comment:
I don't understand why we want to do that ? We specify on model level exactly which model can by set as referrer. If you need this functionality better will be create specify field for this. What do you think ?

Reply:
This was very specific for trying tomaking lazy loading reference fields work ... Probably doesnt make sense for someone else...
removed it from the PR